File: algo-linear-search.py
arrayToSearchThrough = []
for i in range(1,1000):
    arrayToSearchThrough.append(i)

def linearSearch(valueToFind, arrayToSearchThrough):
    for i in range(len(arrayToSearchThrough)):
        if arrayToSearchThrough[i]== valueToFind:
            index = i
    print(index)
    return 0


def globalLinearSearch(valueToFind, arrayToSearchThrough):
    arrayOfIndex=[];
    for i in range(len(arrayToSearchThrough)):
        if arrayToSearchThrough[i]==valueToFind:
            arrayOfIndex.append(i)
        
    
    print(arrayOfIndex)
    return 0

linearSearch(6, arrayToSearchThrough)
globalLinearSearch(4, arrayToSearchThrough)

File: algo-linear-search.js
const arrayToSearchThrough = [];
for (let i = 1; i <= 1000; i++) {
    arrayToSearchThrough.push(i);
}
arrayToSearchThrough.push(4);

function linearSearch(valueToFind, arrayToSearchThrough) {
    let index;
    for (let i = 0; i <= arrayToSearchThrough.length; i++){
        if (arrayToSearchThrough[i]===valueToFind){
            index = i;
        }
    }
    console.log(index);
    return 0
};

function globalLinearSearch(valueToFind, arrayToSearchThrough) {
    let arrayOfIndex=[];
    for (let i = 0; i <= arrayToSearchThrough.length; i++){
        if (arrayToSearchThrough[i]===valueToFind){
            arrayOfIndex.push(i);
        }
    }
    console.log(arrayOfIndex);
    return 0
};


linearSearch(6, arrayToSearchThrough);
globalLinearSearch(4, arrayToSearchThrough);
